pages/common/firefox.md

```diff
@@ -1,31 +1,31 @@
 # firefox
 
 > A free and open source web browser.
-> More information at https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options.
+> More information: <https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options>.
 
-- Open firefox at www.duckduckgo.com:
+- Open Firefox and load www.duckduckgo.com:
 
 `firefox {{https://www.duckduckgo.com}}`
 
-- Open it at a new window:
+- Open a new window:
 
 `firefox --new-window {{https://www.duckduckgo.com}}`
 
-- Open an incognito window:
+- Open a private (incognito) window:
 
 `firefox --private-window`
 
-- Search 'wikipedia' on your default search engine:
+- Open Firefox and search for 'wikipedia' using the default search engine:
```

Comment:
I don't think "Open Firefox" is needed here and the example below. It is pretty much understood.

Reply:
Good point. I took the opportunity to also update the descriptions to disambiguate the words "launch" vs. "open", since "open" was being used for both senses here.